leader-election/DistributedMutex/BlobLeaseManager.cs

```diff
@@ -105,41 +108,45 @@ public async Task<bool> RenewLeaseAsync(string leaseId, CancellationToken token)
         {
             try
             {
-                await this.leaseBlob.RenewLeaseAsync(new AccessCondition { LeaseId = leaseId }, token);
+                if (this.leaseClient == null)
+                {
+                    this.leaseClient = this.leaseBlobClient.GetBlobLeaseClient(leaseId);
+                }
+
+                await this.leaseClient.RenewAsync(cancellationToken: token);
                 return true;
             }
-            catch (StorageException storageException)
+            catch (Azure.RequestFailedException storageException) // use specific exception class for StorageBlob Track2
             {
                 // catch (WebException webException)
-                Trace.TraceError(storageException.Message);
+                Trace.TraceError(storageException.ErrorCode);
 
                 return false;
             }
         }
 
         private async Task CreateBlobAsync(CancellationToken token)
         {
-            await this.leaseBlob.Container.CreateIfNotExistsAsync(token);
-            if (!await this.leaseBlob.ExistsAsync(token))
+            //await this.leaseBlob.Container.CreateIfNotExistsAsync(token);
```

Comment:
Removed commented code - please do this globally.  This will be last time I make this comment, so please look for all comemnted code and remove.  Thanks